mlkem/verify.h

```diff
@@ -6,9 +6,28 @@
 #include <limits.h>
 #include <stddef.h>
 #include <stdint.h>
+#include "arith_native.h"
 #include "cbmc.h"
 #include "params.h"
 
+// If a native backend is used, we include the native implementations from the
+// backend as those are using inline assembly. This guarantees that these
+// gadgets are constant-time, but also allows the compiler to inline them.
+// Otherwise, we use the reference code (verify.c) in a separate compilation
+// unit.
+#if defined(MLKEM_USE_NATIVE_VERIFY)
```

Comment:
If `MLKEM_USE_NATIVE_VERIFY` is set, we don't need to do anything, do we? It would be the responsibility of the native header to define `static inline` verify functions.

Reply:
I'm not sure I follow. What do you want me to change?

Reply:
I'll do it